File: Wordplay/js/9.js
// What are all of the words that have all 5 vowels, in any order?

let main = () => {

    const fs = require("fs");
    let buffer_arry = fs.readFileSync("Wordplay/sowpods.txt", "utf-8");
    let in_file = buffer_arry.split("\n");

    for(let i = 0; i < in_file.length; i++) {

        if(in_file[i].includes("A") == true && in_file[i].includes("E") == true &&
        in_file[i].includes("I") == true && in_file[i].includes("O") == true &&
        in_file[i].includes("U") == true) {

            console.log(in_file[i]);
        };
    };
};

main();

File: Wordplay/python/9.py
# What are all of the words that have all 5 vowels, in any order?
def main():
    in_file = open("../sowpods.txt", "r")
    for words in in_file:
        if("A" in words and "E" in words and "I" in words 
        and "O" in words and "U" in words):
            print(words,end="")

if __name__ == '__main__':
    main()